src/public/js/video-tagging/optional-tags.html

```diff
@@ -69,7 +69,17 @@
             var self = this;
             btn.addEventListener("click", (e) => {
                 let tag = e.currentTarget.id;
-                if (this.tagsState[tag] == "empty" || this.tagsState[tag] == "partial") {
+                if(e.ctrlKey){
+                    let videotagging = document.getElementById('video-tagging');
+                    let index = videotagging.inputtagsarray.indexOf(tag);
+                    if (index !== -1){
+                        videotagging.inputtagsarray.splice(index, 1);
+                        videotagging.optionalTags.colors.splice(index, 1);
+                    }
+                    document.getElementById(tag).remove();
+                    videotagging.checkRemovedTags();
+                }
+                else if (this.tagsState[tag] == "empty" || this.tagsState[tag] == "partial") {
```

Comment:
Update these to use `===` and super nit pick, use single quotes for strings. Good to get in the groove here, as our code linter going forward will complain :slightly_smiling_face: